### PR TITLE
TNSA Example: Link Domain Decomposition

### PR DESCRIPTION
**Pull request by ax3l**
- link domain decomposition for `numprocs` option
- remove outdated warning

### CODE REVIEW

File: Examples/Physics_applications/laser_ion/README.rst [hunk 1]
```diff
@@ -15,12 +15,6 @@ Although laser-ion acceleration requires full 3D modeling for adequate descripti
    This includes spatial and temporal resolution, but also the number of macro-particles per cell representing the target density for proper phase space sampling.
    You will need a computing cluster for adequate resolution of the target density, see comments in the input file.
 
-.. warning::
```
Comment (ax3l) [inline Examples/Physics_applications/laser_ion/README.rst:18]: Outdated since 2023 hackathon.

**Review by RemiLehe — APPROVED** [review]

**Review by n01r — APPROVED** [review]

File: Examples/Physics_applications/laser_ion/README.rst [hunk 2]
```diff
@@ -30,7 +24,12 @@ This example can be run **either** as:
 * **Python** script: ``mpiexec -n 2 python3 PICMI_inputs_2d.py`` or
 * WarpX **executable** using an input file: ``mpiexec -n 2 warpx.2d inputs_2d``
 
-For `MPI-parallel <https://www.mpi-forum.org>`__ runs on computing clusters, change the prefix to ``mpiexec -n <no. of MPI ranks> ...`` or ``srun -n <no. of MPI ranks> ...``, depending on the system and number of MPI ranks you want to allocate.
+.. tip::
+
+   For `MPI-parallel <https://www.mpi-forum.org>`__ runs on computing clusters, change the prefix to ``mpiexec -n <no. of MPI ranks> ...`` or ``srun -n <no. of MPI ranks> ...``, depending on the system and number of MPI ranks you want to allocate.
+
+   The input option ``warpx_numprocs`` / ``warpx.numprocs`` needs to be adjusted for parallel :ref:`domain decomposition <usage_domain_decomposition>`, to match the number of MPI ranks used.
```
Comment (n01r) [inline Examples/Physics_applications/laser_ion/README.rst:31]: Repetition in `warpx.numprocs`

Comment (lucafedeli88) [inline Examples/Physics_applications/laser_ion/README.rst:31]:
```suggestion
   The input option ``warpx_numprocs`` (or, alternatively, the two options ``amr.max_grid_size`` and ``amr.blocking_factor``) needs to be adjusted for parallel :ref:`domain decomposition <usage_domain_decomposition>`, to match the number of MPI ranks used.
```

**Review by lucafedeli88 — CHANGES_REQUESTED** [review]
Thanks for this PR! There's just a small comment by @n01r that needs to be addressed (I've suggested a possible solution)

File: Examples/Physics_applications/laser_ion/README.rst (hunk 2 above)
Comment (ax3l) [reply Examples/Physics_applications/laser_ion/README.rst:31]: It is slightly different: one Python, one inputs file :)

Comment (ax3l) [reply Examples/Physics_applications/laser_ion/README.rst:31]: The two mentions are slightly different: one Python, one inputs file :)

The blocking factor and details are explained in better detail in the link.

Comment (n01r) [reply Examples/Physics_applications/laser_ion/README.rst:31]: Ah, wow, you're right ... sorry - I totally missed the `.` and `_`. 😅

Comment (ax3l) [reply Examples/Physics_applications/laser_ion/README.rst:31]: @lucafedeli88 , I would merge for now - but we can find a more general text we use on all examples :)

**Pull request merged**